src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -42,7 +42,7 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available for this architecture");
```

Comment:
Oh, the `INPUT_FILE` variable is not used. I'll fix that if we decide to use that bam for the end to end test.